src/Main/Settings.cpp

```diff
@@ -0,0 +1,154 @@
+#include "Settings.h"
+
+#include <QDebug>
+#include <QFile>
+#include <QJsonArray>
+#include <QJsonDocument>
+
+#include "stdio.h"
+
+using namespace FOEDAG;
+
+Settings* Settings::m_instance = nullptr;
+Settings::Settings() { fprintf(stderr, "Settings Created\n"); }
```

Comment:
Why not std::cerr ?

Reply:
Old habit from my previous company which was a bit behind the times and more c styled in their code. This is just debug code anyways.